cluster-api/src/main/java/com/sequenceiq/cloudbreak/cluster/service/ClusterComponentConfigProvider.java

```diff
@@ -28,25 +21,41 @@
 import com.sequenceiq.cloudbreak.common.type.ComponentType;
 import com.sequenceiq.cloudbreak.domain.stack.cluster.Cluster;
 import com.sequenceiq.cloudbreak.domain.stack.cluster.ClusterComponent;
+import com.sequenceiq.cloudbreak.domain.stack.cluster.ClusterComponentHistory;
 import com.sequenceiq.cloudbreak.domain.view.ClusterComponentView;
+import com.sequenceiq.cloudbreak.repository.ClusterComponentHistoryRepository;
 import com.sequenceiq.cloudbreak.repository.ClusterComponentRepository;
 import com.sequenceiq.cloudbreak.repository.ClusterComponentViewRepository;
+import org.apache.commons.codec.binary.Base64;
```

Comment:
Please set a correct formatter, see for example line 35 should not be after these imports.
Please use the one included with the project itself